### ISSUE TITLE
Suspiciously narrow test dataset

### ISSUE THREAD

**Issue by BreziTasbi** [ivadomed/lumbar-classification-rsna-challenge-2024#1]
I just noticed that we had a single study_id in the test dataset so a single prediction to make (with 75 different labels nontheless).
This share of the data is supposed to be a 36% of the full test dataset according to the leaderboad headline. (dicom wise?)

It means there must something like 3 different patient in the full testing set. Which looks suspicious considering the amount of data in the training set.
Can someone cross-check my reasoning please ? If i am wrong it means that i didn't undertsand at all how the inference is supposed to be done
![noni](https://github.com/ivadomed/lumbar-classification-rsna-challenge-2024/assets/164916503/a3d1c7d5-a798-402a-940e-e227c105abae)

**Comment by jcohenadad** [comment]
> It means there must something like 3 different patient in the full testing set

if there are 3 patients already, and it represents about 36%, then the full test set is about 9 patients, no? Which is still pretty small... Maybe someone can email the organizers?

**Comment by BreziTasbi** [comment]
actually, to me there are only one patient here since the submition csv seems to expect a single prediction

**Comment by jcohenadad** [comment]
can someone email the organizers or post on the forum to clarify?

**Comment by maxradx** [comment]
Posted on https://www.kaggle.com/competitions/rsna-2024-lumbar-spine-degenerative-classification/discussion/513789
